zerver/lib/actions.py

```diff
@@ -333,6 +368,7 @@ def do_set_realm_name(realm, name):
     return {}
 
 def do_set_realm_restricted_to_domain(realm, restricted):
+    # type: (Realm, bool) -> Dict[None, None]
```

Comment:
Well I tried to look at what looks at their return values and I was not sure that nothing looks at their return values. I'll drop the return statements in a separate commit and see what happens.